mackerel/resource_mackerel_aws_integration_test.go

```diff
@@ -303,7 +303,6 @@ resource "mackerel_aws_integration" "foo" {
 
   ec2 {
     enable               = true
-    role                 = ""
```

Comment:
The `role` field accepts a role ID(`$service_name: $role_name`) or null. Empty string is some kind of pathological case.